sdk/Context/Context.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Sdk\Context;
+
+use OpenTelemetry\Trace as API;
+
+class Context implements API\Context
+{
+    /**
+     * @var ContextKey
+     */
+    private $key;
+
+    /**
+     * @var mixed
+     */
+    private $value;
+
+    /**
+     * @var Context
+     */
+    private $parent;
+
+    /**
+     * Private constructor. All instances should be created via withValue().
+     *
+     * This is heavily influenced by Golang's https://golang.org/pkg/context package
+     */
+    private function __construct(ContextKey $key, $value, ?Context $parent=null)
```

Comment:
We could theoretically use a [class_alias](https://www.php.net/manual/en/function.class-alias.php) and alias a function called `CreateKey` to this class?

That may satisfy : https://github.com/open-telemetry/opentelemetry-php/issues/103

Reply:
Not sure I follow how that would functionally help us?
We currently meet the expectation of the spec via the `ContextKey` public constructor.
I've been using the python and go language libs as reference libs to learn from and neither of them have a _literal_ `CreateKey` method or function, they meet the philosophical aims of the spec via providing the appropriate behavior through other mechanisms, as does the implementation here.

Reply:
Yeah I was just mentioning it so we could easily follow the spec naming conventions; not 100% necessary or potentially valuable.